src/System.Net.Http.WinHttpHandler/src/System/Net/Http/WinHttpTraceHelper.cs

```diff
@@ -11,9 +11,9 @@ namespace System.Net.Http
     internal static class WinHttpTraceHelper
     {
         private const string WinHtpTraceEnvironmentVariable = "WINHTTPHANDLER_TRACE";
-        private static bool s_TraceEnabled;
+        private static readonly bool s_TraceEnabled = ReadTracingEnabled();
 
-        static WinHttpTraceHelper()
+        private static bool ReadTracingEnabled()
```

Comment:
"ReadTracingEnabled" seems confusing for this method name. Why use the "Read" in the name?  Why not somethng like "IsTracingEnabled"?